test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -191,7 +194,30 @@ def prepareTestLoader(self, loader):
 
         def check_loadTestsFromNames(names, module=None):
             rv = old_loadTestsFromNames(names, module)
-            self.check_unknown()
+            test_list = self.__tests_list_copy
+            test_suites = self.__tests_list_copy['__only__ts__']
+            if (len(test_list) > 2) or (len(test_suites) != 0):
+                if 'PBSTestSuite' not in test_suites:
+                    self._du = DshUtils()
+                    ptl_test_dir = self._du.which(exe='pbs_benchpress')
+                    ptl_test_dir = ptl_test_dir.replace('bin/pbs_benchpress',
+                                                        'tests')
+                    user_test_dir = os.environ.get("PTL_TESTS_DIR", None)
+                    if user_test_dir is not None:
+                        if os.path.isdir(user_test_dir):
+                            rv = old_loadTestsFromNames([user_test_dir],
+                                                        module)
+                        else:
+                            logging.error("Invalid dircetory specified in
+                                          PTL_TESTS_DIR")
+                            sys.exit(1)
+                    elif os.path.isdir(ptl_test_dir):
+                        rv = old_loadTestsFromNames([ptl_test_dir], module)
+                    if not self.__need_info:
+                        self.check_unknown()
```

Comment:
What will be an error message if the user runs `pbs_benchpress -i -t UnknownTestSuite` with your changes?

Reply:
So, in pbs_benchpress, I am calling loader plugin and then testinfo plugin. Loader will load all of the test cases in current/User specified/Installed directory and runs check_unknown function which I don't want to call for -i option as it is taken care when testinfo plugin is called.